Formula/tfel.rb

```diff
@@ -0,0 +1,69 @@
+class Tfel < Formula
+  desc "Main repository of TFEL/MFront project"
+  homepage "https://thelfer.github.io/tfel/web/index.html"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/tfel.git", using: :git, branch: "master"
+  stable do
+    url "https://github.com/thelfer/tfel/archive/refs/tags/TFEL-4.0.tar.gz"
+    sha256 "ed3fb2f59c6b8c9896606ef92276f81942433dd5f60d8130ba07c3af80b039e2"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :required
+  depends_on "python@3.9" => :required
```

Comment:
`:required` is not allowed in homebrew core.

Reply:
> `:required` is not allowed in homebrew core.

Hi @SMillerDev, thx for your comments.
Which keyword replaces `:required`? To use the proposed TFEL/MFront code generator, gcc is required under Linux and clang under MacOS.

Reply:
Nothing replaces required. It's a dependency, it's always required.